apps/web/components/booking/CancelBooking.tsx

```diff
@@ -44,6 +49,10 @@ export default function CancelBooking(props: Props) {
   const [loading, setLoading] = useState(false);
   const telemetry = useTelemetry();
   const [error, setError] = useState<string | null>(booking ? null : t("booking_already_cancelled"));
+  const eventType = bookingCancelledEventProps.eventType as EventType | undefined;
+  const cancelReasonField = eventType?.bookingFields
+    ? (eventType.bookingFields as Fields).find((field) => field?.name === "cancelReason")
```

Comment:
Same here. I see know you have already imported eventTypeBookingFields. You should use 
```tsx
const parsedFields = eventType?.bookingFields ? eventTypeBookingFields.parse(eventType.bookingFields) : null;
parsedFields.find(() => ) .....or something similar that suits your needs.
```